lib/createRss.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const _ = require('lodash');
+const ARRAY = [];
```

Comment:
Let's not mix imports and these other constants. Imports should be grouped together at the top, then the rest of your global variables.